Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+pipeline {
+    agent any
+
+    stages {
+        stage('Build') {
+            steps {
```

Comment:
stage build should get the environment ready and not run the code, you can install gems and other necessary processes here